sw/host/tests/chip/sysrst_ctrl/sysrst_ctrl.rs

```diff
@@ -50,3 +50,17 @@ pub fn set_pins(transport: &TransportWrapper, config: &Config, values: u8) -> Re
     }
     Ok(())
 }
+
+/// Read pins: the i-th bit of `values` corresponds to the i-th entry of `config.input_pins`.
+#[allow(dead_code)]
```

Comment:
Please can you remove this attribute in the next commit since it gets used?

Reply:
Done, but I had to turn this into a proper crate between it is used in several tests but not all routines are used in every test.